tests/test_pvc_concurrent_deletion_creation.py

```diff
@@ -0,0 +1,215 @@
+
+    # Switch to default project
+    run_cmd(f'oc project {defaults.ROOK_CLUSTER_NAMESPACE}')
+
+    # Delete project created for the testcase
+    PROJECT.delete(resource_name=TEST_PROJECT)
+
+
+def create_multiple_pvc(pvc_base_name, number_of_pvc, sc_name):
+    """
+    Create PVCs
+
+    Args:
+        pvc_base_name (str): Prefix of PVC name
+        number_of_pvc (int): Number of PVCs to be created
+        sc_name (str): Storage class name
+    """
+    # Parameters for PVC yaml as dict
+    pvc_data = defaults.CSI_PVC_DICT.copy()
+    pvc_data['metadata']['namespace'] = TEST_PROJECT
+    pvc_data['spec']['storageClassName'] = sc_name
+
+    for count in range(1, number_of_pvc + 1):
+        pvc_name = f'{pvc_base_name}{count}'
+        log.info(f'Creating Persistent Volume Claim {pvc_name}')
+        pvc_data['metadata']['name'] = pvc_name
+        PVC_OBJ = PVC(**pvc_data)
+        PVC_OBJ.create()
+        PVC_OBJS.append(PVC_OBJ)
+        log.info(f'Created Persistent Volume Claim {pvc_name}')
+
+
+def run_async(command):
+    """
+    Run command locally and return without waiting for completion
+
+    Args:
+        command (str): The command to run on the system.
+
+    Returns:
+        An open descriptor to be used by the calling function.
+        None on error.
+    """
+    log.info(f"Executing command: {command}")
+    p = subprocess.Popen(
+        command, stdin=subprocess.PIPE, stdout=subprocess.PIPE, shell=True
+    )
+
+    def async_communicate():
+        stdout, stderr = p.communicate()
+        retcode = p.returncode
+        return retcode, stdout, stderr
+
+    p.async_communicate = async_communicate
+    return p
+
+
+@pytest.mark.usefixtures(
+    create_rbd_secret.__name__,
+    create_ceph_block_pool.__name__,
+    create_rbd_storageclass.__name__,
+    test_fixture.__name__
+)
+class TestMultiplePvcConcurrentDeletionCreation(ManageTest):
+    """
+    Test to verify concurrent creation and deletion of multiple PVCs
+    """
+    number_of_pvc = 100
+    pvc_base_name = 'test-pvc'
+    pvc_base_name_new = 'test-pvc-re'
+    initial_pvs = []
+    pvc_objs_initial = []
+    pvc_objs_new = []
+
+    @tier1
+    def test_multiple_pvc_concurrent_creation_deletion(self):
+        """
+        To exercise resource creation and deletion
+        """
+        # Start deleting 100 PVCs
+        command = (
+            f'for i in `seq 1 {self.number_of_pvc}`;do oc delete pvc '
+            f'{self.pvc_base_name}$i -n {TEST_PROJECT};done'
+        )
+        proc = run_async(command)
+        assert proc, (
+            f'Failed to execute command for deleting {self.number_of_pvc} PVCs'
+        )
+
+        # Create 100 new PVCs
+        create_multiple_pvc(
+            self.pvc_base_name_new, self.number_of_pvc, self.sc_obj.name
+        )
+        log.info(f'Created {self.number_of_pvc} new PVCs.')
+        self.pvc_objs_new = PVC_OBJS[:]
+
+        # Verify PVCs are Bound
```

Comment:
helper function under pvc.py

Reply:
Here 'status' attribute of PVC object is used to verify the status. I think this is feasible based on the way this test case is implemented.